.github/workflows/build.yml

```diff
@@ -68,6 +68,9 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v3
         with:
+          # https://github.com/docker/build-push-action/issues/761
+          driver-opts: |
+            image=moby/buildkit:v0.10.6
```

Comment:
The linked PR says this line should fix the failing builds we have experienced for the past few weeks